docs/prisma2-cli.md

```diff
@@ -36,30 +36,30 @@ The following environment variables can be provided:
 
 #### `prisma2 init`
 
-Sets up Prisma (i.e. Photon and/or Lift) via an interactive wizard.
+Sets up Prisma (i.e., Photon and/or Lift) via an interactive wizard. You can specify your database connection and/or create a new database to work with. For a list with currently supported database engines, see the [Lift documentation](./lift/use-only-lift.md).
 
 #### `prisma2 dev`
 
-Starts Prisma [development mode](./development-mode.md).
+Starts Prisma [development mode](./development-mode.md). This opens a web application to interact with your database and defined models. Note that `Prisma Studio` relies on your `prisma.schema` file (and hence the automatically generated Photon client) to provide a fully featured user-interface with CRUD (create / read / update / delete) functionality.
```

Comment:
> This opens a web application to interact with your database and defined models.

Not _quite_ true. It starts the web server for Prisma Studio and prints the URL but doesn't _open_ the web app itself. 

> `Prisma Studio`

I wouldn't use inline code formatting (backticks) for "Prisma Studio" here

> `prisma.schema` file

Let's spell this out as "Prisma schema file" (since it can theoretically be named differently). This is also the official notion/terminology we're using throughout the docs.

> (and hence the automatically generated Photon client) 

I'd omit this since that's rather confusing. Yes, Studio uses Photon under the hood but you don't need to have a Photon generator specified in the Prisma schema to use Studio! (at least this shouldn't be the case, if it is it's a bug 🙈)

Reply:
will be fixed in next commit